UPGRADE-2.0.md

```diff
@@ -0,0 +1,126 @@
+# Upgrading from 1.x to 2.0
+
+API client release 2.0 contained few backward incompatible changes.
+
+This guide will help you upgrade your codebase.
+
+## `UserRecommendation` now returns new format of response in `->getData()`
+
+#### Before
+
+```php
+$recommendation = UserRecommendation::create('user-id', 5, 'scenario', 1.0, 3600);
+
+$response = $matej->request()
+    ->recommendation($recommendation)
+    ->send();
+
+$recommendations = $response->getRecommendation()->getData();
+
+// var_dump($recommendations):
+// array(10) {
+//    [0] =>
+//        string(9) "item_id_1"
+//    [1] =>
+//        string(9) "item_id_2"
+// }
+
+foreach ($recommendations as $recommendedId) {
+    echo $recommendedId; // outputs id of recommended item
+}
+```
+
+#### After
+
+```php
+// Additional properties could now be passed to the UserRecommendation command using constructor or setResponseProperties() method
+$recommendation = UserRecommendation::create('user-id', 5, 'scenario', 1.0, 3600, ['item_title']);
+
+$response = $matej->request()
+    ->recommendation($recommendation)
+    ->send();
+
+$recommendations = $response->getRecommendation()->getData();
+
+// $recommendations now contains array of stdClasse with item_id and other requested properties (item_title in this example)
+
+// var_dump($recommendations):
+// array(2) {
+//     [0] => object(stdClass)#1 (2) {
+//         ["item_id"]  => string(9) "item_id_1"
+//         ["item_title"] => string(5) "title_1"
+//     }
+//     [1] => object(stdClass)#2 (2) {
+//         ["item_id"]  => string(9)  "item_id_2"
+//         ["item_title"] => string(10) "title_2"
+//     }
+// }
+
+foreach ($recommendations as $recommendation) {
+    echo $recommendation->item_id; // outputs id of recommended item
+    echo $recommendation->item_title; // outputs custom item_title property of recommended item
+}
+```
+
+## `UserRecommendation` does not have default filter
+
+If you relied on default filter of UserRecommendation command (`['valid_to >= NOW']`) you must not define it explicitly.
+
+#### Before
+
+```php
+$recommendation = UserRecommendation::create('user-id', 5, 'scenario', 1.0, 3600);
+```
+
+#### After
+
+```php
+$recommendation = UserRecommendation::create('user-id', 5, 'scenario', 1.0, 3600);
+$recommendation->addFilter('valid_to >= NOW()');
```

Comment:
Is this true? Or what is 1:1 equivalent of the old behavior - how should the filter look now?